data/en/structappend.json

```diff
@@ -2,13 +2,14 @@
 	"name":"structAppend",
 	"type":"function",
 	"syntax":"structAppend(destStruct, sourceStruct [, overwriteFlag])",
+	"member": "destStruct.append(sourceStruct [, overwriteFlag])\ndestStruct.putAll(sourceStruct [, overwriteFlag])",
```

Comment:
`putAll` is not a member function. It works because it is falling back on Java's `putAll` method for `Map`, which does not even support the optional second parameter and fails if provided.

Reply:
Oh, okay. Thanks for the hint. I guess we shouldn't refer to java functions doing mostly the same?

Reply:
Since it's not part of CFML and a result of implementation, I would say that it should not be included. It seems that @pfreitag has already merged this PR, but I would be interested in his take on this.

Reply:
Yeah - I'd tend to agree that the java method is technically _undocumented_ and just a side effect of the implementation. I think it would be more suitable as an example to show how you could do it in earlier versions of CF.

Reply:
So kind of related to #812.

Reply:
Sounds good :+1:

So, I make a new pr related to this one with the requested example, right?